cmake/Windows/NSIS.template.in

```diff
@@ -84,14 +83,15 @@ RequestExecutionLevel user    /* RequestExecutionLevel REQUIRED! */
 !include Sections.nsh
 !include Library.nsh
 !include FileFunc.nsh
-!include ${NSISRESOURCES}\UAC.nsh
-!include ${NSISRESOURCES}\IncludeScript_Misc.nsh
+#!include ${NSISRESOURCES}\Include\UAC.nsh

```

Comment:
do elevations still work without the UAC plugin? so it is not really required?
what about x64 and logiclib? not required anymore?

Reply:
They are in the Multiuser plugin.